2022-03-20 14:10:43,126 INFO    Thread-31 :22548 [wandb_setup.py:_flush():75] Loading settings from C:\Users\ashwini.kumar\.config\wandb\settings
2022-03-20 14:10:43,126 INFO    Thread-31 :22548 [wandb_setup.py:_flush():75] Loading settings from wandb\settings
2022-03-20 14:10:43,127 INFO    Thread-31 :22548 [wandb_setup.py:_flush():75] Loading settings from environment variables: {'project': 'uncategorized', 'entity': 'fibonacciash', 'root_dir': 'C:\\Users\\ashwini.kumar\\Time_Series_Modelling', 'sweep_id': 'tgn5xyf2', 'run_id': 'm60gt6zk', 'sweep_param_path': 'C:\\Users\\ashwini.kumar\\Time_Series_Modelling\\wandb\\sweep-tgn5xyf2\\config-m60gt6zk.yaml'}
2022-03-20 14:10:43,127 INFO    Thread-31 :22548 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-20 14:10:43,132 INFO    Thread-31 :22548 [wandb_init.py:_log_setup():405] Logging user logs to C:\Users\ashwini.kumar\Time_Series_Modelling\wandb\run-20220320_141043-m60gt6zk\logs\debug.log
2022-03-20 14:10:43,133 INFO    Thread-31 :22548 [wandb_init.py:_log_setup():406] Logging internal logs to C:\Users\ashwini.kumar\Time_Series_Modelling\wandb\run-20220320_141043-m60gt6zk\logs\debug-internal.log
2022-03-20 14:10:43,133 INFO    Thread-31 :22548 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000014261899EC8>
2022-03-20 14:10:43,134 INFO    Thread-31 :22548 [wandb_init.py:init():439] calling init triggers
2022-03-20 14:10:43,134 INFO    Thread-31 :22548 [wandb_init.py:init():443] wandb.init called with sweep_config: {'input_chunk_length': 48, 'layer_widths': 512, 'output_chunk_length': 12}
config: {}
2022-03-20 14:10:43,134 INFO    Thread-31 :22548 [wandb_init.py:init():492] starting backend
2022-03-20 14:10:43,134 INFO    Thread-31 :22548 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2022-03-20 14:10:43,144 INFO    Thread-31 :22548 [backend.py:ensure_launched():219] starting backend process...
2022-03-20 14:10:43,223 INFO    Thread-31 :22548 [backend.py:ensure_launched():225] started backend process with pid: 18068
2022-03-20 14:10:43,223 INFO    Thread-31 :22548 [wandb_init.py:init():501] backend started and connected
2022-03-20 14:10:43,230 INFO    Thread-31 :22548 [wandb_run.py:_config_callback():992] config_cb None None {'input_chunk_length': 48, 'layer_widths': 512, 'output_chunk_length': 12}
2022-03-20 14:10:43,231 INFO    Thread-31 :22548 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-20 14:10:43,231 INFO    Thread-31 :22548 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-20 14:10:43,231 INFO    Thread-31 :22548 [wandb_init.py:init():565] updated telemetry
2022-03-20 14:10:43,233 INFO    Thread-31 :22548 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-20 14:10:44,792 INFO    Thread-31 :22548 [wandb_run.py:_on_init():1759] communicating current version
2022-03-20 14:10:44,895 INFO    Thread-31 :22548 [wandb_run.py:_on_init():1763] got version response 
2022-03-20 14:10:44,895 INFO    Thread-31 :22548 [wandb_init.py:init():625] starting run threads in backend
2022-03-20 14:10:45,050 INFO    Thread-31 :22548 [wandb_run.py:_console_start():1733] atexit reg
2022-03-20 14:10:45,051 INFO    Thread-31 :22548 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-20 14:10:45,052 INFO    Thread-31 :22548 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-20 14:10:45,614 INFO    Thread-31 :22548 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-20 14:10:45,615 INFO    Thread-31 :22548 [wandb_init.py:init():664] run started, returning control to user process
2022-03-20 14:14:22,993 INFO    Thread-31 :22548 [wandb_run.py:finish():1539] finishing run fibonacciash/uncategorized/m60gt6zk
2022-03-20 14:14:22,993 INFO    Thread-31 :22548 [jupyter.py:save_history():429] not saving jupyter history
2022-03-20 14:14:22,994 INFO    Thread-31 :22548 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-20 14:14:22,994 INFO    Thread-31 :22548 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-03-20 14:14:22,994 INFO    Thread-31 :22548 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-20 14:14:22,995 INFO    Thread-31 :22548 [wandb_run.py:_restore():1674] restore
2022-03-20 14:14:23,737 INFO    Thread-31 :22548 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
  media_count: 5
}
pusher_stats {
  uploaded_bytes: 661960
  total_bytes: 661960
}

2022-03-20 14:14:24,150 INFO    Thread-31 :22548 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
  media_count: 5
}
pusher_stats {
  uploaded_bytes: 661960
  total_bytes: 661960
}

2022-03-20 14:14:24,925 INFO    Thread-31 :22548 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
  media_count: 5
}
pusher_stats {
  uploaded_bytes: 661960
  total_bytes: 676483
}

2022-03-20 14:14:25,040 INFO    Thread-31 :22548 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
  media_count: 5
}
pusher_stats {
  uploaded_bytes: 661960
  total_bytes: 676483
}

2022-03-20 14:14:25,154 INFO    Thread-31 :22548 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
  media_count: 5
}
pusher_stats {
  uploaded_bytes: 661960
  total_bytes: 676483
}

2022-03-20 14:14:25,285 INFO    Thread-31 :22548 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
  media_count: 5
}
pusher_stats {
  uploaded_bytes: 661960
  total_bytes: 676483
}

2022-03-20 14:14:25,388 INFO    Thread-31 :22548 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
  media_count: 5
}
pusher_stats {
  uploaded_bytes: 661960
  total_bytes: 676483
}

2022-03-20 14:14:25,493 INFO    Thread-31 :22548 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
  media_count: 5
}
pusher_stats {
  uploaded_bytes: 676483
  total_bytes: 676483
}

2022-03-20 14:14:25,627 INFO    Thread-31 :22548 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
  media_count: 5
}
pusher_stats {
  uploaded_bytes: 676483
  total_bytes: 676483
}

2022-03-20 14:14:25,747 INFO    Thread-31 :22548 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
  media_count: 5
}
pusher_stats {
  uploaded_bytes: 676483
  total_bytes: 676483
}

2022-03-20 14:14:25,864 INFO    Thread-31 :22548 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
  media_count: 5
}
pusher_stats {
  uploaded_bytes: 676483
  total_bytes: 676483
}

2022-03-20 14:14:25,973 INFO    Thread-31 :22548 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
  media_count: 5
}
pusher_stats {
  uploaded_bytes: 676483
  total_bytes: 676483
}

2022-03-20 14:14:26,081 INFO    Thread-31 :22548 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
  media_count: 5
}
pusher_stats {
  uploaded_bytes: 676483
  total_bytes: 676483
}

2022-03-20 14:14:27,419 INFO    Thread-31 :22548 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
  media_count: 5
}
pusher_stats {
  uploaded_bytes: 676483
  total_bytes: 676483
}

2022-03-20 14:14:28,231 INFO    Thread-31 :22548 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  media_count: 5
}
pusher_stats {
  uploaded_bytes: 676483
  total_bytes: 676483
}
local_info {
}

2022-03-20 14:14:29,406 INFO    Thread-31 :22548 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-20 14:14:29,406 INFO    Thread-31 :22548 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-20 14:14:29,413 INFO    Thread-31 :22548 [wandb_run.py:_footer_sync_info():2822] logging synced files
